tests/Exception/RequestExceptionTest.php

```diff
@@ -130,21 +130,6 @@ public function testExceptionMessageIgnoresEmptyBody()
         self::assertStringEndsWith('response', $e->getMessage());
     }
 
-    public function testCreatesExceptionWithoutPrintableBody()
```

Comment:
I've added an issue for this https://github.com/guzzle/guzzle/issues/2765

Reply:
We already discussed this a few months ago and decided this was not a bug.